Sources/SourceControl/GitRepository.swift

```diff
@@ -600,6 +600,17 @@ private class GitFileSystemView: FileSystem {
         }
     }
     
+    func isExecutableFile(_ path: AbsolutePath) -> Bool {
+        do {
+            if let entry = try getEntry(path), entry.type == .executableBlob {
+                return true
+            }
+            return false
+        } catch {
+            return false
+        }
```

Comment:
I think this can be simplified to:
```swift
if let entry = try? getEntry(path), entry.type == .executableBlob {
    return true
}
return false
```